model/model_test.go

```diff
@@ -0,0 +1,157 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+
```

Comment:
nit: can you remove white space :)

Reply:
goimports and gofmt wants imports from third-party to be separated from local imports (imports from the same repo). I think that will not pass the gofmt check.

Reply:
interesting i have gofmt enabled in goland and it never complained :)